### DISCUSSION TITLE
v1.3 Release Scoping: Features Graduating to Standard

### DISCUSSION THREAD

**Discussion by robscott** [kubernetes-sigs/gateway-api#3404] [General]
This discussion is similar to https://github.com/kubernetes-sigs/gateway-api/discussions/3403, but somewhat simpler. We'll track every GEP that could potentially graduate to standard in this release, and then look for owners to volunteer to drive them forward. Each top level comment will describe a GEP + what is required to take it to standard. To volunteer as a lead for that GEP, please add a comment in the thread below the GEP description.

As a reminder, each GEP that graduates to standard creates a new slot for a GEP to graduate from experimental (#3403), so if you really want a new experimental feature, volunteering to lead an experimental feature to standard will help. 

Note that inclusion in the list below is not a guarantee that a GEP will graduate in v1.3, just an acknowledgement that it theoretically could with sufficient effort and consensus.

**Comment by jakebennert** [comment]
I'd like to graduate #3171 Percentage-Based Request Mirroring to standard in this release.

The API for this GEP has been completed (#3283), but there still exists implementation and docs work, which I will be happy to own.

**Comment by candita** [comment]
I would like to help [BackendTLSPolicy](https://gateway-api.sigs.k8s.io/geps/gep-1897/) graduate to standard in v1.3.  

The [API implementation ](https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1alpha3/backendtlspolicy_types.go) has been completed (and amended), and docs work is complete.  There are at least ~~three~~ four implementations of BackendTLSPolicy: Envoy Gateway, Contour, ~~and~~ NGINX, and  Traefik. ~~is also rumored to support BackendTLSPolicy.~~  Normative [conformance test](https://github.com/kubernetes-sigs/gateway-api/pull/3212) is nearing completion.

**Reply by nmengin** [reply]
Hello @candita,

I confirm that Traefik v3.2 brings BackendTLSPolicy support 😄

**Comment by guicassolato** [comment]
I would like to move Policy Attachment ([GEP-2648](https://gateway-api.sigs.k8s.io/geps/gep-2648/) and [GEP-2649](https://gateway-api.sigs.k8s.io/geps/gep-2649/)) to Standard.

Even though parts of those GEPs have been moved to Experimental only a couple versions ago, and others  are technically still flagged as Provisional, not only they have been long enough debated since [GEP-713](https://gateway-api.sigs.k8s.io/geps/gep-713/) but also, in practice, several implementations have been quite committed to the pattern already.

**Reply by mikemorris** [reply]
The only objection I have to the policy attachment pattern is the terminology "ancestor" in `policyAncestorStatus` for what I believe is intended to communicate something more like policy affected/impacted status, including direct and indirect references (but not necessarily ancestral - such as resources within a targeted namespace or routes attached to a targeted gateway).

**Comment by candita** [comment]
I would like to see TLSroute move to Standard.  TLSroute predates the requirement for existing GEPS in order to become Experimental, though we could add one if deemed necessary.  Over time [several implementations have added TLSRoute](https://gateway-api.sigs.k8s.io/implementations/v1.1/#tlsroute) based only the experimental API:
Cilium, Contour, Envoy Gateway, Istio, and nginx-gateway-fabric. 
I should add that Red Hat needs the TLSRoute (for passthrough) in its Gateway API integrations.

A writeup here has 6 upvotes: https://github.com/kubernetes-sigs/gateway-api/discussions/3410
